test/test_create_spoken_forms.py

```diff
@@ -4,9 +4,40 @@
     # Only include this when we're running tests
 
     import itertools
+    from typing import IO, Callable
 
     from talon import actions
 
+    import core.abbreviate
+    import core.user_settings
+
+    core.abbreviate.abbreviations_list = {"source": "src", "whats app": "WhatsApp"}
+
+    # we need to replace the track_csv_list decorator for unit tests.
```

Comment:
I think instead of doing this we should call the decorated function in the fake implementation of `talon.resource.watch`.